app/packages/grits-net-meteor/client/templates/grits_dataTable.jade

```diff
@@ -24,25 +27,25 @@ template(name='gritsDataTable')
                 th          {{_ "gritsDataTable.explore.col3-heading"}}
                 th          {{_ "gritsDataTable.explore.col4-heading"}}
             tbody
-              with paths
-                each paths
-                  tr.pathTableRow(data-id='{{this._id}}')
-                    td          {{this.origin._id}}
-                    td.dtHidden {{getNodeName this.origin}}
-                    td.dtHidden {{getNodeCity this.origin}}
-                    td.dtHidden {{getNodeState this.origin}}
-                    td.dtHidden {{getNodeCountry this.origin}}
-                    td          {{this.destination._id}}
-                    td.dtHidden {{getNodeName this.destination}}
-                    td.dtHidden {{getNodeCity this.destination}}
-                    td.dtHidden {{getNodeState this.destination}}
-                    td.dtHidden {{getNodeCountry this.destination}}
-                    td          {{this.occurrences}}
-                    td          {{this.throughput}}
-                      .circle(style='background-color: {{getPathThroughputColor this}}')
-              tr
-                td(colspan='12')
-                  span(style='font-style: italic;') No data
+              each paths
+                tr.pathTableRow(data-id='{{this._id}}')
+                  td          {{this.origin._id}}
+                  td.dtHidden {{getNodeName this.origin}}
+                  td.dtHidden {{getNodeCity this.origin}}
+                  td.dtHidden {{getNodeState this.origin}}
+                  td.dtHidden {{getNodeCountry this.origin}}
+                  td          {{this.destination._id}}
+                  td.dtHidden {{getNodeName this.destination}}
+                  td.dtHidden {{getNodeCity this.destination}}
+                  td.dtHidden {{getNodeState this.destination}}
+                  td.dtHidden {{getNodeCountry this.destination}}
+                  td          {{this.occurrences}}
+                  td          {{this.throughput}}
+                    .circle(style='background-color: {{getPathThroughputColor this}}')
+              else
```

Comment:
This is cool - I didn't realize that you could use an `else` to handle empty collections